lib/content-services/src/lib/content-node-share/content-node-share.dialog.ts

```diff
@@ -99,7 +105,17 @@ export class ShareDialogComponent implements OnInit, OnDestroy {
             } else {
                 this.sharedId = properties['qshare:sharedId'];
                 this.isFileShared = true;
-                this.updateForm();
+
+                const expiryDate = this.updateForm();
+                if (expiryDate !== null && expiryDate !== undefined) {
```

Comment:
I think this can be simplified to something like: 
```ts
this.isExpiryDateToggleChecked  = this.isLinkWithExpiryDate = expiryDate !== null && expiryDate !== undefined;
this.isLinkWithExpiryDate ? this.time.enable() : this.time.disable();
```